hudi-common/src/test/java/org/apache/hudi/common/table/read/TestFileGroupRecordBuffer.java

```diff
@@ -84,8 +84,6 @@ class TestFileGroupRecordBuffer {
   private final FileGroupReaderSchemaHandler schemaHandler =
       mock(FileGroupReaderSchemaHandler.class);
   private HoodieTableMetaClient hoodieTableMetaClient = mock(HoodieTableMetaClient.class);
-  private Option<String> partitionNameOverrideOpt = Option.empty();
```

Comment:
same comment as above.

Reply:
This was an unused variable so I removed it